lua/caramba/search.lua

```
@@ -858,4 +945,314 @@ M.index_workspace_keyword = function(callback)
+  if embeddings_exist then
+    local stat = vim.loop.fs_stat(paths.embeddings_path)
+    if stat then
+      result.info.embeddings_size = stat.size
+    end
+  end
+  
+  return result
+end
+
+-- Remove all embeddings for a file
+local function remove_file_embeddings(filepath)
```

Luacheck (1.2.0)

[warning] unused function 'remove_file_embeddings'

(W211)